code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -520,14 +529,18 @@ var/global/list/robot_modules = list(
 
 /obj/item/robot_module/janitor/New()
 	..()
-	src.modules += new /obj/item/crowbar/robotic(src)
-	src.modules += new /obj/item/device/flash(src)
 	src.modules += new /obj/item/soap/nanotrasen(src)
 	src.modules += new /obj/item/storage/bag/trash(src)
 	src.modules += new /obj/item/mop(src)
 	src.modules += new /obj/item/device/lightreplacer/advanced(src)
 	src.modules += new /obj/item/reagent_containers/glass/bucket(src) // a hydroponist's bucket
 	src.modules += new /obj/item/matter_decompiler(src) // free drone remains for all
+	src.modules += new /obj/item/taperoll/engineering(src) // To enable 'borgs to telegraph danger visually.
+	src.modules += new /obj/item/inflatable_dispenser(src) // To enable 'borgs to protect Crew from danger in direct hazards.
```

Comment:
Not on the janitor borg

Reply:
What do you mean not on the Janitorial 'borg? it has the same duties as all of the other 'borgs and just is geared towards cleaning the Station, it still is expected to serve/protect crew just like all of the others?

Reply:
OK So after matt brought up a point in Discord about how inflatables could be abused, I decided to remove them from the PR, but something like them might be needed in the future to enable 'borgs to seal breaches/protect Crew from suffocation.

Inflatables are a negatory

Reply:
This I hope is resolved, reopen if not. I'm also going to resolve the dittos since they're v. related to this.

Reply:
As said in the new comment, the 'borg inflatables have a delay on being placed down, so the exploit explained to me can't actually be done. I did lower the number of inflatables they can have on them at any time from 10 walls to 5 and from 5 doors to 3, which should only really enable them to block a single airlock at a time without charging.

I feel this is a p. extreme nerf to them for the extreme benefit of allowing Crew to be able to depend on 'borgs to protect them in atmos hazards.